src/Sylius/Component/Grid/FieldTypes/DatetimeFieldType.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Grid\FieldTypes;
+
+use Sylius\Component\Grid\DataExtractor\DataExtractorInterface;
+use Sylius\Component\Grid\Definition\Field;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class DatetimeFieldType implements FieldTypeInterface
+{
+    /**
+     * @var DataExtractorInterface
+     */
+    private $dataExtractor;
+
+    /**
+     * @param DataExtractorInterface $dataExtractor
+     */
+    public function __construct(DataExtractorInterface $dataExtractor)
+    {
+        $this->dataExtractor = $dataExtractor;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function render(Field $field, $data)
+    {
+        /** @var \DateTime $value */
+        $value = $this->dataExtractor->get($field, $data);
+        if (null === $value) {
```

Comment:
Sure! I didn't add it, because I wasn't sure where we are now with fields validation... it would be good to validate ``format`` variable to be proper datetime format :smile: but for now - validation of value type :+1: